man/systemd-system.conf.xml

```diff
@@ -171,6 +171,126 @@
         effect if a hardware watchdog is not available.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IOCostQOSPath=</varname></term>
+        <listitem><para> Configure the absolute path to device to be configured
+        in io.cost.qos file of the root cgroup.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostQOSCtrl=</varname></term>
```

Comment:
Not fond of such abbreviations, "Ctrl"? There's nothing wrong with just spelling out "Control"